coffee_boat/captain.py

```diff
@@ -75,6 +75,17 @@ def launch_ship(self):
     .. note::
        This function must be called before your init your SparkContext!"""
     self._raise_if_running()
+
+    # Doing sketchy things with the gateway
```

Comment:
What's the implication of this?

Reply:
So we need to shut down the existing JVM process since that's (currently) the only way the PYTHON_PATH can be updated.

Reply:
No way to change environment variables for the JVM process? That's too bad, though entirely sensible.

Reply:
So the problem is it's cached inside a val inside of a class somewhere. We could maybe do something to change it, but I think that would be even sketchier than just bouncing the JVM.